xformers/components/attention/csrc/attention.cpp

```diff
@@ -1,5 +1,18 @@
 #include <torch/types.h>
 
+// If we are in a Windows environment, we need to define
+// initialization functions for the _custom_ops extension.
+// For PyMODINIT_FUNC to work, we need to include Python.h
+// https://github.com/pytorch/vision/blob/main/torchvision/csrc/vision.cpp#L17
+// Fixes error LNK2001: unresolved external symbol PyInit__C
+#if defined(_WIN32)
+#include <Python.h>
+PyMODINIT_FUNC PyInit__C(void) {
+  // No need to do anything.
+  return NULL;
+}
+#endif // defined(_WIN32)
```

Comment:
Thanks for this! This is a longstanding issue with torchvision as well which we haven't found a better solution so far.